README.md

```diff
@@ -49,9 +49,9 @@ Here is a more detailed [roadmap](https://github.com/OrchardCMS/OrchardCore/wiki
 
 ### Visual Studio
 
-- Download Visual Studio 2022 (any edition) from https://www.visualstudio.com/downloads/
-- Open `OrchardCore.sln` and wait for Visual Studio to restore all Nuget packages
-- Ensure `OrchardCore.Cms.Web` is the startup project and run it
+- Download Visual Studio 2022 (v17.5+) from https://www.visualstudio.com/downloads/
```

Comment:
Not necessary `v17.5+` I have `17.4.3`

Reply:
Maybe you are not using .NET 7?

https://dotnet.microsoft.com/en-us/download/dotnet/7.0

![image](https://user-images.githubusercontent.com/24724371/219509921-362767be-e92c-4be8-b243-022af3fd80cb.png)

Reply:
I see `7.0.3` has been released 3 days ago